source/install/ee-install.rst

```diff
@@ -181,11 +181,23 @@ Use this command to upload a new license or to replace an existing license with
 .. note::
 
   If you upload the license via the CLI using  ``mattermost license upload``, you need to restart the Mattermost server after uploading. Additionally, if you're running a cluster, the license file needs to be uploaded to every node. See `our documentation for more information on the command line tools <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-license-upload>`__.
+  
+License key storage
+^^^^^^^^^^^^^^^^^^^^
+
+Once you've uploaded your license key to your Mattermost server it's stored in your SQL database at ``mattermost.Licenses``. You can check what keys are on your server by running ``select * from mattermost.Licenses;``.
 
 **Using the System Console**
 
 1. Open **System Console > About > Edition and License** (or **System Console > OTHER > Edition and License** in versions prior to 5.12).
 2. Click **Remove Enterprise License and Downgrade Server**. This clears the license from the server and refreshes the System Console.
 3. Upload the new license key file.
 
+Removing an E20 or E10 license key will not remove the configuration for Enterprise settings, however these features will not function until an E10 or E20 license key is applied. 
+
+.. note::
+
+  - When you apply an E20 license key to a previously E10-licensed server, the E10 features will retain their configuration settings in E20. 
+  - When you apply an E10 license to a previously E20-licensed server, the E20 features will retain their configuration but will no longer be accessible for use.
+
```

Comment:
Please add:  Please note that when using High Availability to ensure all servers in the cluster have the E20 license properly installed to prevent multi-node clusters from failing. An E20 license is required for High Availability to work.

Maybe this can go in at line 197.

Reply:
I've added it and agree with the placement.